emgapi/models.py

```diff
@@ -502,14 +502,25 @@ class Publication(models.Model):
         db_column='PUBLISHED_YEAR', blank=True, null=True,
         help_text='Published year')
     pub_type = models.CharField(
-        db_column='PUB_TYPE', max_length=150, blank=True, null=True)
+        db_column='PUB_TYPE', max_length=300, blank=True, null=True)
 
     objects = PublicationManager()
 
     class Meta:
         db_table = 'PUBLICATION'
         ordering = ('pubmed_id',)
 
+    def save(self, *args, **kwargs):
+        for field in self._meta.fields:
+            if isinstance(field, models.TextField) or isinstance(field, models.CharField):
```

Comment:
We could do something more fancy here, but I would defer that for the time being.

Reply:
Fair. In a future update where we upgrade the db engine, we will probably switch all these artificially max-length'd fields over to `TextField`s anyway.